.github/workflows/push-build.yaml

```diff
@@ -18,13 +18,21 @@ permissions:
 jobs:
   push-image-to-container-registry:
     runs-on: ubuntu-22.04
+    permissions:
+      packages: write
     if: github.repository == 'rook/rook'
     steps:
       - name: checkout
         uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
         with:
           fetch-depth: 0
 
+      - name: consider debugging
```

Comment:
If testing is done, we can remove this debugging step, right?

Reply:
This is a good point. Allowing tmate in the workflow that has access to image push secrets is a risk for our build and release pipeline.